### PR TITLE
Revert PR#6839 (coexistence-chat-calls-presence.md)

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7434.

I agree with the author of this issue. I think the following PR was not correct:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/6839

The following sentence is talking about in-tenant interoperability and federation:
"If the Skype for Business account of the originator is homed on-premises, that user can't use the Teams client for in-tenant interoperability or for federation. That user can only use the Skype for Business client for interoperability and federation."
If an sfb user is homed on-premises and upgrade mode is Island, he can't use the Teams client to communicate with an in-tenant SfBO user (SfB* upgrade mode) or federated user.

The tables show routing in all scenarios, not just interoperability.

@RuiPereiraTabares @bjwhalen @jambirk @cichur, maybe you can kindly share your thoughts here. Thank you so much!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by bjwhalen** [comment]
can someone summarize what is the actual change that is being proposed? there are many tables that look the same to so it's hard to parse out.  Looking at the live page as of July 6, 2021, i'm not seeing anything obvious that needs to change. Federation and interop for any user in a Teams client, who has sfb onprem is not possible and this is what i see in all tables on this page. because the relevant row in each table says "not possible" under recipient.  I'm not seeing anything that needs to change but the topic is complex, and the docs are dense, so if i missed something, pls call it out.

**Comment by dariomws** [comment]
@bjwhalen sure.

Table 1a:
| <br/><br/> Mode | Originator <br/><br/> Client | <br/><br/> SfB&nbsp;homed |<br/><br/>Route-->| Recipient <br/><br/> Islands  |
|--- |--- |--- |--- |--- |
| Islands | Teams | On-prem | &boxv;|**Not Possible**|
| | | | | |

It should be "Teams" instead of "Not possible". It's taking about in-tenant routing, not interoperability. Messages sent to Islands users will always route to the same client from which they were sent.

Table 1c:
| <br/><br/> Mode | Originator <br/><br/> Client | <br/><br/> SfB&nbsp;homed |<br/><br/>Route-->| Recipient <br/><br/> Teams Only |
|--- |--- |--- |--- |--- |
| Islands | Teams | On-prem | &boxv;|**Not Possible**|
| | | | | |

**Comment by bjwhalen** [comment]
On the one hand i get what you are saying: within same tenant Teams always routes to Teams as long as recipient is not in an SFB mode.  But we should only change 1a, not 1b. In table 1B, the recipient mode is SFB*. So in that case, the recipient won't receive anything in Teams, hence Not Possible is correct for that row in 1B.  We should change1A but not 1B. 

The larger issue is the overall presentation/structure/organization of this information is confusing and potentially misleading.  There is a lot of redundancy in the presentations of information, and the key differences are hard to discern. that's beyond the scope of this change here, but i think that's the real long term solution

**Comment by dariomws** [comment]
@bjwhalen I meant table 1a (Island to Island) and 1c (Island to Teams Only).

I agree that this article is confusing and misleading. It needs to be rethought and rewritten.

**Comment by dariomws** [comment]
Closing this PR as it was updated here:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/commit/b8cda4a377f37a2cf6c7f1147bca87185e9ccf3e#diff-ccaa1fd683a23f24a261e109bde659811359807cf406ef793d798e3b61b217c1
